lib/stateful/animation.dart

```diff
@@ -0,0 +1,288 @@
+import 'dart:async';
+import 'dart:math' as math;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+class StatefulAnimation extends StatefulWidget {
+  StatefulAnimation({
+    Key key,
+  }): super(key: key);
+
+  @override
+  _StatefulAnimationState createState() => _StatefulAnimationState();
+}
+
+class _StatefulAnimationState extends State<StatefulAnimation> {
+  ValueNotifier<Color> _colorListenable;
+  Timer _timer;
+
+  @override
+  void initState() {
+    super.initState();
+    _colorListenable = ValueNotifier<Color>(Colors.green[100]);
+    int color = 100;
+    _timer = Timer.periodic(const Duration(milliseconds: 900), (Timer timer) {
```

Comment:
Would it make sense to instead use an AnimationController.repeat + tween, so that we benefit from TickerMode and are closer to common use-cases?

Reply:
No, because this only ticks every 900ms, not every 16ms. But it would be good to take TickerMode into account.